src/pages/Validators/ValidatorsTable.tsx

```diff
@@ -163,7 +163,7 @@ function ValidatorHeaderCell({
   }
 }
 
-export type ValidatorCellProps = {
+type ValidatorCellProps = {
```

Comment:
why do we have two `type ValidatorCellProps`? one in src/pages/Validators/DelegationValidatorsTable.tsx:L224?

Reply:
cuz we have two tables. one is validator table for all nodes, another is delegation validator table. before, since they don't differentiate in props, i re-used them, but now need to separate them up.